doc/introduction.txt

```diff
@@ -165,7 +165,7 @@ Note: There is no short term plan to support multi-node computation.
 Theano Vision State
 ===================
 
-Here is the state of that vision as of November 15th, 2017 (after Theano 1.0.0):
+Here is the state of that vision as of December 1st, 2017 (after Theano 1.0.1):
```

Comment:
I don't think this has to be updated since the vision hasn't changed and it would be nice for people to know since when that has changed.

Also, since we say "support for one year", it's nice to know since when.